packages/shared/src/borsh-schemas.ts

```diff
@@ -1,6 +1,15 @@
-import { field, vec } from "@dao-xyz/borsh";
+import { BinaryReader, BinaryWriter, field, vec } from "@dao-xyz/borsh";
 import BigNumber from "bignumber.js";
-import { BigNumberSerializer } from "@namada/types";
+
+export const BigNumberSerializer = {
```

Comment:
What does it do to copy BigNumberSerializer to shared?

Reply:
I wanted to remove dependency from types  :) In fturue it  wouldbe  nice to plan a  bit what is dependant on what.